activerecord/test/cases/adapter_test.rb

```diff
@@ -14,6 +14,19 @@ def setup
       @connection.materialize_transactions
     end
 
+    def test_type_map_is_ractor_shareable
+      # This is testing internals. Please feel free to remove this test
+      # or change it when internals change. The point is to make sure
+      # the type map is Ractor shareable.
+      skip unless defined?(Ractor)
+      @connection.tables.each do |table|
+        @connection.columns(table).each do |column|
+          type = @connection.lookup_cast_type_from_column(column)
+          assert Ractor.shareable?(type)
```

Comment:
f this sort of test will become common, maybe we ought to have some `assert_ractor_sharable` method? So that we'd at least get the object `inspect` or something on failure?

Reply:
Ya, I think so too. But I think lets extract after we find it to be a pattern. I'm not sure how far this experiment is going to go.

Reply:
Even if it's just at the bottom of `adapter_test`, a simple `assert_ractor_shareable` that does:

```ruby
def assert_ractor_shareable(obj)
  assert  Ractor.shareable?(obj), -> { "Expected #{obj} to be shareable, but it wasn't" }
end
```

So that the failure message is decent.